src/LoggerListeners.cpp

```diff
@@ -46,40 +46,40 @@
 		}
 
 		#ifdef _DEBUG
-			virtual void Log(AString a_Message, cLogger::eLogLevel a_LogLevel) override
+			virtual void Log(std::string_view a_Message, eLogLevel a_LogLevel) override
 			{
 				Super::Log(a_Message, a_LogLevel);
 				// In a Windows Debug build, output the log to debug console as well:
-				OutputDebugStringA(a_Message.c_str());
+				OutputDebugStringA(AString(a_Message).c_str());
```

Comment:
Note we can avoid a further string copy here if we guarantee that `a_Message` has a terminating null character. However, is only used in debug builds anyway so I'm not too bothered.